src/redux/books/booksSlice.js

```diff
@@ -1,38 +1,68 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import fetchData from '../../utils/fetchData';
 
 const initialState = {
```

Comment:
- [ ] Kindly check your slice and redux store setup. It seems that it doesn't work properly. When I'm adding a new book, in the store I can see only the last one. And it would be great if the book it's stored with the `name`, `author` and `category` property 💯